.github/workflows/php-stan.yaml

```diff
@@ -35,20 +51,18 @@ jobs:
                 }
               }
             }
-      - name: Get composer cache directory
-        id: composer-cache
-        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
-      - name: Cache dependencies
-        uses: actions/cache@v3
-        with:
-          path: ${{ steps.composer-cache.outputs.dir }}
-          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
-          restore-keys: ${{ runner.os }}-composer-
-      - name: Install dependencies
-        uses: nick-invision/retry@v2
-        with:
-          timeout_minutes: 5
-          max_attempts: 5
-          command: composer install --prefer-dist --no-interaction --no-progress
+
+      - name: Install composer dependencies
+        uses: ramsey/composer-install@v2
+
       - name: Run PHPStan
-        run: composer phpstan
+        run: ${{ inputs.command }}
+        env:
+          PHPSTAN_ERROR_FILE: ${{ inputs.error_file }}
+
+      - name: phpstan errors artifact
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v2
```

Comment:
They have a `v3` if you want to bump it.

Reply:
Thanks, updated!